packages/web-components/src/components/va-memorable-date/test/va-memorable-date.e2e.ts

```diff
@@ -765,7 +814,7 @@ describe('va-memorable-date', () => {
       expect(date.getAttribute('error')).toEqual(`year-range`);
     });
 
-    it('uswds v3 does month validation without required prop', async () => {
+    it.skip('uswds v3 does month validation without required prop', async () => {
```

Comment:
Why is this being skipped?

Reply:
This was originally being skipped because of this issue where month wasn't able to show an error since day had a valid value: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2123

I have modified the test now though so that there is no default day value and month will show an error